test/avocado/selenium-machines-disks.py

```diff
@@ -19,3 +45,141 @@ def testDiskInfo(self):
+        self.create_vm(name, wait=True)
+        pool_name = self.prepare_disk()
+
+        self.click(self.wait_css('#vm-{}-disks'.format(name), cond=clickable))
+        self.click(self.wait_css('#vm-{}-disks-adddisk'.format(name), cond=clickable))
+        Select(self.wait_css('#vm-{}-disks-adddisk-new-select-pool'.format(name))).select_by_value(pool_name[1])
+        self.send_keys(self.wait_css('#vm-{}-disks-adddisk-new-name'.format(name)), 'qcow2disk')
+        self.check_box(self.wait_css('#vm-{}-disks-adddisk-new-permanent'.format(name), cond=clickable))
+        self.click(self.wait_css('#vm-{}-disks-adddisk-dialog-add'.format(name), cond=clickable))
+        self.storage_pool[pool_name[1]].append('qcow2disk')
+        self.wait_dialog_disappear()
+        self.wait_css('#vm-{}-disks-vda-device'.format(name))
+
+        self.click(self.wait_css('#vm-{}-disks-adddisk'.format(name), cond=clickable))
+        Select(self.wait_css('#vm-{}-disks-adddisk-new-select-pool'.format(name))).select_by_value(pool_name[0])
+        self.send_keys(self.wait_css('#vm-{}-disks-adddisk-new-name'.format(name)), 'raw2disk')
+        Select(self.wait_css('#vm-{}-disks-adddisk-new-diskfileformat'.format(name))).select_by_value('raw')
+        self.click(self.wait_css('#vm-{}-disks-adddisk-dialog-add'.format(name), cond=clickable))
+        self.storage_pool[pool_name[0]].append('raw2disk')
+        self.wait_dialog_disappear()
+        self.wait_css('#vm-{}-disks-vdb-device'.format(name))
+
+        self.click(self.wait_css('#vm-{}-disks-adddisk'.format(name), cond=clickable))
+        self.click(self.wait_css('#vm-{}-disks-adddisk-useexisting'.format(name), cond=clickable))
+        Select(self.wait_css('#vm-{}-disks-adddisk-existing-select-pool'.format(name))).select_by_value(pool_name[0])
+        self.click(self.wait_css('#vm-{}-disks-adddisk-dialog-add'.format(name, cond=clickable)))
+        self.wait_dialog_disappear()
+        self.wait_css('#vm-{}-disks-vdc-device'.format(name))
+
+        self.click(self.wait_css('#vm-{}-disks-adddisk'.format(name), cond=clickable))
+        self.click(self.wait_css('#vm-{}-disks-adddisk-useexisting'.format(name), cond=clickable))
+        Select(self.wait_css('#vm-{}-disks-adddisk-existing-select-pool'.format(name))).select_by_value(pool_name[1])
+        Select(self.wait_css('#vm-{}-disks-adddisk-existing-select-volume'.format(name))).select_by_value('mypooldisk2')
+        self.check_box(self.wait_css('#vm-{}-disks-adddisk-existing-permanent'.format(name), cond=clickable))
+        self.click(self.wait_css('#vm-{}-disks-adddisk-dialog-add'.format(name, cond=clickable)))
+        self.wait_dialog_disappear()
+        self.wait_css('#vm-{}-disks-vdd-device'.format(name))
+
+        self.click(self.wait_css('#vm-{}-off'.format(name), cond=clickable))
+        self.wait_css('#vm-{}-off'.format(name), cond=invisible)
+
+        # It seems that infoRecord component was adjusted to not use table,
```

Comment:
Please don't have comments on the old code. The html + css change consistently and tests should be adjusted all the time, having comments of how the code looked in the past won't help anyone.

Reply:
OK, i will delete them